src/components/sectionList/download/DownloadDialogContent.tsx

```diff
@@ -0,0 +1,109 @@
+                })}
+            </ModalTitle>
+            <ModalContent>
+                <form className={css.downloadForm}>
+                    <UIField
+                        label="Compression"
+                        className={css.horizontalRadio}
+                    >
+                        <Field<string | undefined>
+                            name="compression"
+                            component={RadioFieldFF}
+                            label="Zip"
+                            type="radio"
+                            value={'zip'}
+                        />
+                        <Field<string | undefined>
+                            name="compression"
+                            component={RadioFieldFF}
+                            value="gz"
+                            label="Gzip"
+                            type="radio"
+                        />
+                        <Field<string | undefined>
+                            name="compression"
+                            component={RadioFieldFF}
+                            value="uncompressed"
+                            label="Uncompressed"
+                            type="radio"
+                        />
+                    </UIField>
+                    <UIField
+                        label={i18n.t('{{model}} to include', {
+                            model: section.titlePlural,
+                        })}
+                        className={css.horizontalRadio}
+                    >
+                        <Field<string | undefined>
+                            name="filter"
+                            component={RadioFieldFF}
+                            label={i18n.t('All ({{number}})', {
+                                number: totalQuery.data || 0,
+                            })}
+                            type="radio"
+                            value={'all'}
+                        />
+                        {selectedModels.size > 0 && (
+                            <Field<string | undefined>
+                                name="filter"
+                                component={RadioFieldFF}
+                                label={i18n.t('Only selected ({{number}})', {
+                                    number: selectedModels.size,
+                                })}
+                                type="radio"
+                                value={'selected'}
+                            />
+                        )}
+                        <Field<string | undefined>
+                            name="filter"
+                            label={i18n.t(
+                                'Only objects that match selected filters ({{number}})',
+                                { number: totalWithFiltersQuery.data || 0 }
+                            )}
+                            component={RadioFieldFF}
+                            value="filters"
+                            type="radio"
+                        />
```

Comment:
Very minor: Would it be possible & make sense to disable this option and hide the count when no filter has been selected?

Reply:
Fixed. Now disabled and added a tooltip for why it's disabled